Make sure that the Boost headers are actually found, since they are
referenced in the kdepimlibs headers. 
Patch by Alex Turbov, see https://bugs.kde.org/show_bug.cgi?id=306323

--- a/KdepimLibsConfig.cmake.in.org	2012-08-13 12:46:24.000000000 +0400
+++ b/KdepimLibsConfig.cmake.in	2012-09-06 08:53:53.000000000 +0400
@@ -15,7 +15,7 @@
 set(KDEPIMLIBS_DBUS_INTERFACES_DIR "@KDEPIMLIBS_DBUS_INTERFACES_DIR@")
 set(KDEPIMLIBS_DBUS_SERVICES_DIR   "@KDEPIMLIBS_DBUS_SERVICES_DIR@")
 set(KDEPIMLIBS_INCLUDE_DIR         "@KDEPIMLIBS_INCLUDE_DIR@")
-set(KDEPIMLIBS_INCLUDE_DIRS        "@KDEPIMLIBS_INCLUDE_DIR@" "@KDEPIMLIBS_INCLUDE_DIR@/KDE")
+set(KDEPIMLIBS_INCLUDE_DIRS        "@KDEPIMLIBS_INCLUDE_DIR@" "@KDEPIMLIBS_INCLUDE_DIR@/KDE" "@Boost_INCLUDE_DIR@")
 set(KDEPIMLIBS_LIB_DIR             "@KDEPIMLIBS_LIB_DIR@")
 set(KDEPIMLIBS_BIN_DIR             "@KDEPIMLIBS_BIN_DIR@")
 set(KDEPIMLIBS_LIBEXEC_DIR         "@KDEPIMLIBS_LIBEXEC_DIR@")
